File: client/src/config/DayConfig.ts
class DayConfig {
    private static _dic:Dictionary<DayConfig> = new Dictionary<DayConfig>();
    private static _arr:Array<DayConfig> = new Array<DayConfig>();

	 public Id :number ;
     public MarketType :number ;
     public Day :string ;
     public IsOpen :number ;
    
    

    
    public constructor(){

    }


    public static get Dic():Dictionary<DayConfig>{
        return  DayConfig._dic;
    }
    public static get Arr():Array<DayConfig>{
        return DayConfig._arr;
    }
    public static parse(json:any[]){

        for(var i = 0; i < json.length; i++){
            let config:DayConfig = <DayConfig>json[i];
            DayConfig._arr.push(config);
            DayConfig._dic.add(config.Id,config);
        }
    }
}

File: tools/conf/output/go/DayConfig.go
package conf

import (
	//"bufio"
	"encoding/json"
	"fmt"
	//"io"
	"io/ioutil"
	"os"
	//"strings"
)

type DayConfigElem struct {
	Id int
	MarketType int
	Day string
	IsOpen int
	Des string
	
}

type DayConfig0 struct {
	Dic map[int]*DayConfigElem
	Arr []*DayConfigElem
}

var (
	DayConfig DayConfig0
)

func loadDayConfig(path string) {
	file, err := os.Open(path)
	defer func() {
		if file != nil {
			file.Close()
		}
	}()
	if err != nil {
		panic(fmt.Sprint("load config error, path=", path))
	}

	DayConfig.Dic = make(map[int]*DayConfigElem)
	DayConfig.Arr = make([]*DayConfigElem, 0, 32)

	bt,err  :=  ioutil.ReadAll(file)
	json.Unmarshal(bt, &DayConfig.Arr)
	for _,conf := range DayConfig.Arr{
		DayConfig.Dic[conf.Id] = conf
	}
	
}
